tox.ini

```diff
@@ -1,14 +1,11 @@
 [tox]
 envlist =
-    py{27,35,36,37}-django{111,20,21,22,master},
-    py{36,37}-django30,
+    py{36,37,38}-django{111,django22,django30,master},
```

Comment:
@ulgens This line should be

```ini
py{36,37,38}-django{111,22,30,master},
```

Otherwise it is not consistent with the deps section

Reply:
I'll check that, thanks.

Reply:
I realized that the tests don't test anything:

https://travis-ci.org/github/graphql-python/graphene-django/jobs/663374517?utm_medium=notification&utm_source=github_status

Only django111 and djangomaster do tests.

Reply:
@ulgens when I enable tox, this happens:

https://github.com/ganwell/graphene-django/runs/563812647

Reply:
Is it rebased version of that branch?

Reply:
No I just took your branch and fixed tox:

https://github.com/ganwell/graphene-django/pull/1/commits/8bb11c8f65225cc6331433845e7bef9f2ee100e8

Reply:
This is the result on my local. I know something is wrong in tox config but i have a busy business schedule nowadays, couldn't check it on Travis, sorry. I'll try to allocate some time asap.

``` (graphene)  ~/workspace/gql/graphene-django   graphql-next  make tests
py.test graphene_django --cov=graphene_django -vv
=========================================================================================================== test session starts ============================================================================================================
platform linux -- Python 3.8.2, pytest-5.4.1, py-1.8.1, pluggy-0.13.1 -- /home/ulgens/.pyenv/versions/3.8.2/envs/graphene/bin/python3.8
cachedir: .pytest_cache
django: settings: django_test_settings (from ini)
rootdir: /home/ulgens/Dropbox/workspace/gql/graphene-django, inifile: pytest.ini
plugins: django-3.8.0, cov-2.8.1
collected 266 items                                                                                                                                                                                                                        

graphene_django/debug/tests/test_query.py::test_should_query_field PASSED                                                                                                                                                            [  0%]
graphene_django/debug/tests/test_query.py::test_should_query_nested_field PASSED                                                                                                                                                     [  0%]
graphene_django/debug/tests/test_query.py::test_should_query_list PASSED                                                                                                                                                             [  1%]
graphene_django/debug/tests/test_query.py::test_should_query_connection PASSED                                                                                                                                                       [  1%]
graphene_django/debug/tests/test_query.py::test_should_query_connectionfilter PASSED                                                                                                                                                 [  1%]
graphene_django/filter/tests/test_fields.py::test_filter_explicit_filterset_arguments PASSED                                                                                                                                         [  2%]
graphene_django/filter/tests/test_fields.py::test_filter_shortcut_filterset_arguments_list PASSED                                                                                                                                    [  2%]
graphene_django/filter/tests/test_fields.py::test_filter_shortcut_filterset_arguments_dict PASSED                                                                                                                                    [  3%]
graphene_django/filter/tests/test_fields.py::test_filter_explicit_filterset_orderable PASSED                                                                                                                                         [  3%]
graphene_django/filter/tests/test_fields.py::test_filter_explicit_filterset_not_orderable PASSED                                                                                                                                     [  3%]
graphene_django/filter/tests/test_fields.py::test_filter_shortcut_filterset_extra_meta PASSED                                                                                                                                        [  4%]
graphene_django/filter/tests/test_fields.py::test_filter_shortcut_filterset_context PASSED                                                                                                                                           [  4%]
graphene_django/filter/tests/test_fields.py::test_filter_filterset_information_on_meta PASSED                                                                                                                                        [  4%]
graphene_django/filter/tests/test_fields.py::test_filter_filterset_information_on_meta_related PASSED                                                                                                                                [  5%]
graphene_django/filter/tests/test_fields.py::test_filter_filterset_class_filter_fields_exception PASSED                                                                                                                              [  5%]
graphene_django/filter/tests/test_fields.py::test_filter_filterset_class_information_on_meta PASSED                                                                                                                                  [  6%]
graphene_django/filter/tests/test_fields.py::test_filter_filterset_class_information_on_meta_related PASSED                                                                                                                          [  6%]
graphene_django/filter/tests/test_fields.py::test_filter_filterset_related_results PASSED                                                                                                                                            [  6%]
graphene_django/filter/tests/test_fields.py::test_global_id_field_implicit PASSED                                                                                                                                                    [  7%]
graphene_django/filter/tests/test_fields.py::test_global_id_field_explicit PASSED                                                                                                                                                    [  7%]
graphene_django/filter/tests/test_fields.py::test_filterset_descriptions PASSED                                                                                                                                                      [  7%]
graphene_django/filter/tests/test_fields.py::test_global_id_field_relation PASSED                                                                                                                                                    [  8%]
graphene_django/filter/tests/test_fields.py::test_global_id_multiple_field_implicit PASSED                                                                                                                                           [  8%]
graphene_django/filter/tests/test_fields.py::test_global_id_multiple_field_explicit PASSED                                                                                                                                           [  9%]
graphene_django/filter/tests/test_fields.py::test_global_id_multiple_field_implicit_reverse PASSED                                                                                                                                   [  9%]
graphene_django/filter/tests/test_fields.py::test_global_id_multiple_field_explicit_reverse PASSED                                                                                                                                   [  9%]
graphene_django/filter/tests/test_fields.py::test_filter_filterset_related_results_with_filter PASSED                                                                                                                                [ 10%]
graphene_django/filter/tests/test_fields.py::test_recursive_filter_connection PASSED                                                                                                                                                 [ 10%]
graphene_django/filter/tests/test_fields.py::test_should_query_filter_node_limit PASSED                                                                                                                                              [ 10%]
graphene_django/filter/tests/test_fields.py::test_order_by_is_perserved PASSED                                                                                                                                                       [ 11%]
graphene_django/filter/tests/test_fields.py::test_annotation_is_preserved PASSED                                                                                                                                                     [ 11%]
graphene_django/filter/tests/test_fields.py::test_annotation_with_only PASSED                                                                                                                                                        [ 12%]
graphene_django/filter/tests/test_fields.py::test_node_get_queryset_is_called PASSED                                                                                                                                                 [ 12%]
graphene_django/filter/tests/test_fields.py::test_integer_field_filter_type PASSED                                                                                                                                                   [ 12%]
graphene_django/filter/tests/test_fields.py::test_other_filter_types PASSED                                                                                                                                                          [ 13%]
graphene_django/filter/tests/test_fields.py::test_filter_filterset_based_on_mixin PASSED                                                                                                                                             [ 13%]
graphene_django/rest_framework/tests/test_multiple_model_serializers.py::test_create_schema PASSED                                                                                                                                   [ 13%]
graphene_django/rest_framework/tests/test_mutation.py::test_write_only_field PASSED                                                                                                                                                  [ 14%]
graphene_django/rest_framework/tests/test_mutation.py::test_write_only_field_using_extra_kwargs PASSED                                                                                                                               [ 14%]
graphene_django/rest_framework/tests/test_mutation.py::test_read_only_fields PASSED                                                                                                                                                  [ 15%]
graphene_django/rest_framework/tests/test_mutation.py::test_model_add_mutate_and_get_payload_success PASSED                                                                                                                          [ 15%]
graphene_django/rest_framework/tests/test_mutation.py::test_model_update_mutate_and_get_payload_success PASSED                                                                                                                       [ 15%]
graphene_django/rest_framework/tests/test_mutation.py::test_model_partial_update_mutate_and_get_payload_success PASSED                                                                                                               [ 16%]
graphene_django/rest_framework/tests/test_mutation.py::test_model_invalid_update_mutate_and_get_payload_success PASSED                                                                                                               [ 16%]
graphene_django/rest_framework/tests/test_mutation.py::test_perform_mutate_success PASSED                                                                                                                                            [ 16%]
graphene_django/tests/test_fields.py::TestDjangoListField::test_only_django_object_types PASSED                                                                                                                                      [ 17%]
graphene_django/tests/test_fields.py::TestDjangoListField::test_only_import_paths PASSED                                                                                                                                             [ 17%]
graphene_django/tests/test_fields.py::TestDjangoListField::test_non_null_type PASSED                                                                                                                                                 [ 18%]
graphene_django/tests/test_fields.py::TestDjangoListField::test_get_django_model PASSED                                                                                                                                              [ 18%]
graphene_django/tests/test_fields.py::TestDjangoListField::test_list_field_default_queryset PASSED                                                                                                                                   [ 18%]
graphene_django/tests/test_fields.py::TestDjangoListField::test_override_resolver PASSED                                                                                                                                             [ 19%]
graphene_django/tests/test_fields.py::TestDjangoListField::test_nested_list_field PASSED                                                                                                                                             [ 19%]
graphene_django/tests/test_fields.py::TestDjangoListField::test_override_resolver_nested_list_field PASSED                                                                                                                           [ 19%]
graphene_django/tests/test_query.py::test_should_query_only_fields PASSED                                                                                                                                                            [ 20%]
graphene_django/tests/test_query.py::test_should_query_simplelazy_objects PASSED                                                                                                                                                     [ 20%]
graphene_django/tests/test_query.py::test_should_query_well PASSED                                                                                                                                                                   [ 21%]
graphene_django/tests/test_query.py::test_should_query_postgres_fields PASSED                                                                                                                                                        [ 21%]
graphene_django/tests/test_query.py::test_should_node PASSED                                                                                                                                                                         [ 21%]
graphene_django/tests/test_query.py::test_should_query_onetoone_fields PASSED                                                                                                                                                        [ 22%]
graphene_django/tests/test_query.py::test_should_query_connectionfields PASSED                                                                                                                                                       [ 22%]
graphene_django/tests/test_query.py::test_should_keep_annotations PASSED                                                                                                                                                             [ 22%]
graphene_django/tests/test_query.py::test_should_query_node_filtering PASSED                                                                                                                                                         [ 23%]
graphene_django/tests/test_query.py::test_should_query_node_filtering_with_distinct_queryset PASSED                                                                                                                                  [ 23%]
graphene_django/tests/test_query.py::test_should_query_node_multiple_filtering PASSED                                                                                                                                                [ 24%]
graphene_django/tests/test_query.py::test_should_enforce_first_or_last PASSED                                                                                                                                                        [ 24%]
graphene_django/tests/test_query.py::test_should_error_if_first_is_greater_than_max PASSED                                                                                                                                           [ 24%]
graphene_django/tests/test_query.py::test_should_error_if_last_is_greater_than_max PASSED                                                                                                                                            [ 25%]
graphene_django/tests/test_query.py::test_should_query_promise_connectionfields PASSED                                                                                                                                               [ 25%]
graphene_django/tests/test_query.py::test_should_query_connectionfields_with_last PASSED                                                                                                                                             [ 25%]
graphene_django/tests/test_query.py::test_should_query_connectionfields_with_manager PASSED                                                                                                                                          [ 26%]
graphene_django/tests/test_query.py::test_should_query_dataloader_fields PASSED                                                                                                                                                      [ 26%]
graphene_django/tests/test_query.py::test_should_handle_inherited_choices PASSED                                                                                                                                                     [ 27%]
graphene_django/tests/test_query.py::test_proxy_model_support PASSED                                                                                                                                                                 [ 27%]
graphene_django/tests/test_query.py::test_should_resolve_get_queryset_connectionfields PASSED                                                                                                                                        [ 27%]
graphene_django/tests/test_query.py::test_should_preserve_prefetch_related PASSED                                                                                                                                                    [ 28%]
graphene_django/tests/test_query.py::test_should_preserve_annotations PASSED                                                                                                                                                         [ 28%]
graphene_django/forms/tests/test_converter.py::test_should_unknown_django_field_raise_exception PASSED                                                                                                                               [ 28%]
graphene_django/forms/tests/test_converter.py::test_should_date_convert_date PASSED                                                                                                                                                  [ 29%]
graphene_django/forms/tests/test_converter.py::test_should_time_convert_time PASSED                                                                                                                                                  [ 29%]
graphene_django/forms/tests/test_converter.py::test_should_date_time_convert_date_time PASSED                                                                                                                                        [ 30%]
graphene_django/forms/tests/test_converter.py::test_should_char_convert_string PASSED                                                                                                                                                [ 30%]
graphene_django/forms/tests/test_converter.py::test_should_email_convert_string PASSED                                                                                                                                               [ 30%]
graphene_django/forms/tests/test_converter.py::test_should_slug_convert_string PASSED                                                                                                                                                [ 31%]
graphene_django/forms/tests/test_converter.py::test_should_url_convert_string PASSED                                                                                                                                                 [ 31%]
graphene_django/forms/tests/test_converter.py::test_should_choice_convert_string PASSED                                                                                                                                              [ 31%]
graphene_django/forms/tests/test_converter.py::test_should_base_field_convert_string PASSED                                                                                                                                          [ 32%]
graphene_django/forms/tests/test_converter.py::test_should_regex_convert_string PASSED                                                                                                                                               [ 32%]
graphene_django/forms/tests/test_converter.py::test_should_uuid_convert_string PASSED                                                                                                                                                [ 33%]
graphene_django/forms/tests/test_converter.py::test_should_integer_convert_int PASSED                                                                                                                                                [ 33%]
graphene_django/forms/tests/test_converter.py::test_should_boolean_convert_boolean PASSED                                                                                                                                            [ 33%]
graphene_django/forms/tests/test_converter.py::test_should_nullboolean_convert_boolean PASSED                                                                                                                                        [ 34%]
graphene_django/forms/tests/test_converter.py::test_should_float_convert_float PASSED                                                                                                                                                [ 34%]
graphene_django/forms/tests/test_converter.py::test_should_decimal_convert_float PASSED                                                                                                                                              [ 34%]
graphene_django/forms/tests/test_converter.py::test_should_multiple_choice_convert_connectionorlist PASSED                                                                                                                           [ 35%]
graphene_django/forms/tests/test_converter.py::test_should_manytoone_convert_connectionorlist PASSED                                                                                                                                 [ 35%]
graphene_django/forms/tests/test_mutation.py::test_needs_form_class PASSED                                                                                                                                                           [ 36%]
graphene_django/forms/tests/test_mutation.py::test_has_output_fields PASSED                                                                                                                                                          [ 36%]
graphene_django/forms/tests/test_mutation.py::test_has_input_fields PASSED                                                                                                                                                           [ 36%]
graphene_django/forms/tests/test_mutation.py::test_mutation_error_camelcased PASSED                                                                                                                                                  [ 37%]
graphene_django/forms/tests/test_mutation.py::FormMutationTests::test_form_invalid_form PASSED                                                                                                                                       [ 37%]
graphene_django/forms/tests/test_mutation.py::FormMutationTests::test_form_valid_input PASSED                                                                                                                                        [ 37%]
graphene_django/forms/tests/test_mutation.py::ModelFormMutationTests::test_custom_return_field_name PASSED                                                                                                                           [ 38%]
graphene_django/forms/tests/test_mutation.py::ModelFormMutationTests::test_default_input_meta_fields PASSED                                                                                                                          [ 38%]
graphene_django/forms/tests/test_mutation.py::ModelFormMutationTests::test_default_meta_fields PASSED                                                                                                                                [ 39%]
graphene_django/forms/tests/test_mutation.py::ModelFormMutationTests::test_exclude_fields_input_meta_fields PASSED                                                                                                                   [ 39%]
graphene_django/forms/tests/test_mutation.py::ModelFormMutationTests::test_model_form_mutation_creates_new PASSED                                                                                                                    [ 39%]
graphene_django/forms/tests/test_mutation.py::ModelFormMutationTests::test_model_form_mutation_mutate_existing PASSED                                                                                                                [ 40%]
graphene_django/forms/tests/test_mutation.py::ModelFormMutationTests::test_model_form_mutation_mutate_invalid_form PASSED                                                                                                            [ 40%]
graphene_django/forms/tests/test_mutation.py::ModelFormMutationTests::test_return_field_name_is_camelcased PASSED                                                                                                                    [ 40%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_unknown_rest_framework_field_raise_exception PASSED                                                                                                        [ 41%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_char_convert_string PASSED                                                                                                                                 [ 41%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_email_convert_string PASSED                                                                                                                                [ 42%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_slug_convert_string PASSED                                                                                                                                 [ 42%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_url_convert_string PASSED                                                                                                                                  [ 42%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_choice_convert_enum PASSED                                                                                                                                 [ 43%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_choice_convert_string_if_enum_disabled PASSED                                                                                                              [ 43%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_base_field_convert_string PASSED                                                                                                                           [ 43%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_regex_convert_string PASSED                                                                                                                                [ 44%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_uuid_convert_string PASSED                                                                                                                                 [ 44%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_model_convert_field PASSED                                                                                                                                 [ 45%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_date_time_convert_datetime PASSED                                                                                                                          [ 45%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_date_convert_date PASSED                                                                                                                                   [ 45%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_time_convert_time PASSED                                                                                                                                   [ 46%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_integer_convert_int PASSED                                                                                                                                 [ 46%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_boolean_convert_boolean PASSED                                                                                                                             [ 46%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_float_convert_float PASSED                                                                                                                                 [ 47%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_decimal_convert_float PASSED                                                                                                                               [ 47%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_list_convert_to_list PASSED                                                                                                                                [ 48%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_list_serializer_convert_to_list PASSED                                                                                                                     [ 48%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_dict_convert_dict PASSED                                                                                                                                   [ 48%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_duration_convert_string PASSED                                                                                                                             [ 49%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_file_convert_string PASSED                                                                                                                                 [ 49%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_filepath_convert_string PASSED                                                                                                                             [ 50%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_ip_convert_string PASSED                                                                                                                                   [ 50%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_image_convert_string PASSED                                                                                                                                [ 50%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_json_convert_jsonstring PASSED                                                                                                                             [ 51%]
graphene_django/rest_framework/tests/test_field_converter.py::test_should_multiplechoicefield_convert_to_list_of_enum PASSED                                                                                                         [ 51%]
graphene_django/rest_framework/tests/test_mutation.py::test_needs_serializer_class PASSED                                                                                                                                            [ 51%]
graphene_django/rest_framework/tests/test_mutation.py::test_has_fields PASSED                                                                                                                                                        [ 52%]
graphene_django/rest_framework/tests/test_mutation.py::test_has_input_fields PASSED                                                                                                                                                  [ 52%]
graphene_django/rest_framework/tests/test_mutation.py::test_exclude_fields PASSED                                                                                                                                                    [ 53%]
graphene_django/rest_framework/tests/test_mutation.py::test_nested_model PASSED                                                                                                                                                      [ 53%]
graphene_django/rest_framework/tests/test_mutation.py::test_mutate_and_get_payload_success PASSED                                                                                                                                    [ 53%]
graphene_django/rest_framework/tests/test_mutation.py::test_mutate_and_get_payload_error PASSED                                                                                                                                      [ 54%]
graphene_django/rest_framework/tests/test_mutation.py::test_model_mutate_and_get_payload_error PASSED                                                                                                                                [ 54%]
graphene_django/rest_framework/tests/test_mutation.py::test_mutation_error_camelcased PASSED                                                                                                                                         [ 54%]
graphene_django/rest_framework/tests/test_mutation.py::test_invalid_serializer_operations PASSED                                                                                                                                     [ 55%]
graphene_django/tests/test_command.py::test_generate_json_file_on_call_graphql_schema PASSED                                                                                                                                         [ 55%]
graphene_django/tests/test_command.py::test_json_files_are_canonical PASSED                                                                                                                                                          [ 56%]
graphene_django/tests/test_command.py::test_generate_graphql_file_on_call_graphql_schema PASSED                                                                                                                                      [ 56%]
graphene_django/tests/test_converter.py::test_should_unknown_django_field_raise_exception PASSED                                                                                                                                     [ 56%]
graphene_django/tests/test_converter.py::test_should_date_time_convert_string PASSED                                                                                                                                                 [ 57%]
graphene_django/tests/test_converter.py::test_should_date_convert_string PASSED                                                                                                                                                      [ 57%]
graphene_django/tests/test_converter.py::test_should_time_convert_string PASSED                                                                                                                                                      [ 57%]
graphene_django/tests/test_converter.py::test_should_char_convert_string PASSED                                                                                                                                                      [ 58%]
graphene_django/tests/test_converter.py::test_should_text_convert_string PASSED                                                                                                                                                      [ 58%]
graphene_django/tests/test_converter.py::test_should_email_convert_string PASSED                                                                                                                                                     [ 59%]
graphene_django/tests/test_converter.py::test_should_slug_convert_string PASSED                                                                                                                                                      [ 59%]
graphene_django/tests/test_converter.py::test_should_url_convert_string PASSED                                                                                                                                                       [ 59%]
graphene_django/tests/test_converter.py::test_should_ipaddress_convert_string PASSED                                                                                                                                                 [ 60%]
graphene_django/tests/test_converter.py::test_should_file_convert_string PASSED                                                                                                                                                      [ 60%]
graphene_django/tests/test_converter.py::test_should_image_convert_string PASSED                                                                                                                                                     [ 60%]
graphene_django/tests/test_converter.py::test_should_file_path_field_convert_string PASSED                                                                                                                                           [ 61%]
graphene_django/tests/test_converter.py::test_should_auto_convert_id PASSED                                                                                                                                                          [ 61%]
graphene_django/tests/test_converter.py::test_should_uuid_convert_id PASSED                                                                                                                                                          [ 62%]
graphene_django/tests/test_converter.py::test_should_auto_convert_duration PASSED                                                                                                                                                    [ 62%]
graphene_django/tests/test_converter.py::test_should_positive_integer_convert_int PASSED                                                                                                                                             [ 62%]
graphene_django/tests/test_converter.py::test_should_positive_small_convert_int PASSED                                                                                                                                               [ 63%]
graphene_django/tests/test_converter.py::test_should_small_integer_convert_int PASSED                                                                                                                                                [ 63%]
graphene_django/tests/test_converter.py::test_should_big_integer_convert_int PASSED                                                                                                                                                  [ 63%]
graphene_django/tests/test_converter.py::test_should_integer_convert_int PASSED                                                                                                                                                      [ 64%]
graphene_django/tests/test_converter.py::test_should_boolean_convert_boolean PASSED                                                                                                                                                  [ 64%]
graphene_django/tests/test_converter.py::test_should_nullboolean_convert_boolean PASSED                                                                                                                                              [ 65%]
graphene_django/tests/test_converter.py::test_field_with_choices_convert_enum PASSED                                                                                                                                                 [ 65%]
graphene_django/tests/test_converter.py::test_field_with_grouped_choices PASSED                                                                                                                                                      [ 65%]
graphene_django/tests/test_converter.py::test_field_with_choices_gettext PASSED                                                                                                                                                      [ 66%]
graphene_django/tests/test_converter.py::test_field_with_choices_collision PASSED                                                                                                                                                    [ 66%]
graphene_django/tests/test_converter.py::test_field_with_choices_convert_enum_false PASSED                                                                                                                                           [ 66%]
graphene_django/tests/test_converter.py::test_should_float_convert_float PASSED                                                                                                                                                      [ 67%]
graphene_django/tests/test_converter.py::test_should_manytomany_convert_connectionorlist PASSED                                                                                                                                      [ 67%]
graphene_django/tests/test_converter.py::test_should_manytomany_convert_connectionorlist_list PASSED                                                                                                                                 [ 68%]
graphene_django/tests/test_converter.py::test_should_manytomany_convert_connectionorlist_connection PASSED                                                                                                                           [ 68%]
graphene_django/tests/test_converter.py::test_should_manytoone_convert_connectionorlist PASSED                                                                                                                                       [ 68%]
graphene_django/tests/test_converter.py::test_should_onetoone_reverse_convert_model PASSED                                                                                                                                           [ 69%]
graphene_django/tests/test_converter.py::test_should_postgres_array_convert_list PASSED                                                                                                                                              [ 69%]
graphene_django/tests/test_converter.py::test_should_postgres_array_multiple_convert_list PASSED                                                                                                                                     [ 69%]
graphene_django/tests/test_converter.py::test_should_postgres_hstore_convert_string PASSED                                                                                                                                           [ 70%]
graphene_django/tests/test_converter.py::test_should_postgres_json_convert_string PASSED                                                                                                                                             [ 70%]
graphene_django/tests/test_converter.py::test_should_postgres_range_convert_list PASSED                                                                                                                                              [ 71%]
graphene_django/tests/test_converter.py::test_generate_enum_name PASSED                                                                                                                                                              [ 71%]
graphene_django/tests/test_forms.py::test_global_id_valid PASSED                                                                                                                                                                     [ 71%]
graphene_django/tests/test_forms.py::test_global_id_invalid PASSED                                                                                                                                                                   [ 72%]
graphene_django/tests/test_forms.py::test_global_id_multiple_valid PASSED                                                                                                                                                            [ 72%]
graphene_django/tests/test_forms.py::test_global_id_multiple_invalid PASSED                                                                                                                                                          [ 72%]
graphene_django/tests/test_forms.py::test_global_id_none PASSED                                                                                                                                                                      [ 73%]
graphene_django/tests/test_forms.py::test_global_id_none_optional PASSED                                                                                                                                                             [ 73%]
graphene_django/tests/test_schema.py::test_should_raise_if_no_model PASSED                                                                                                                                                           [ 74%]
graphene_django/tests/test_schema.py::test_should_raise_if_model_is_invalid PASSED                                                                                                                                                   [ 74%]
graphene_django/tests/test_schema.py::test_should_map_fields_correctly PASSED                                                                                                                                                        [ 74%]
graphene_django/tests/test_schema.py::test_should_map_only_few_fields PASSED                                                                                                                                                         [ 75%]
graphene_django/tests/test_types.py::test_django_interface PASSED                                                                                                                                                                    [ 75%]
graphene_django/tests/test_types.py::test_django_get_node PASSED                                                                                                                                                                     [ 75%]
graphene_django/tests/test_types.py::test_django_objecttype_map_correct_fields PASSED                                                                                                                                                [ 76%]
graphene_django/tests/test_types.py::test_django_objecttype_with_node_have_correct_fields PASSED                                                                                                                                     [ 76%]
graphene_django/tests/test_types.py::test_django_objecttype_with_custom_meta PASSED                                                                                                                                                  [ 77%]
graphene_django/tests/test_types.py::test_schema_representation PASSED                                                                                                                                                               [ 77%]
graphene_django/tests/test_types.py::test_django_objecttype_only_fields PASSED                                                                                                                                                       [ 77%]
graphene_django/tests/test_types.py::test_django_objecttype_fields PASSED                                                                                                                                                            [ 78%]
graphene_django/tests/test_types.py::test_django_objecttype_only_fields_and_fields PASSED                                                                                                                                            [ 78%]
graphene_django/tests/test_types.py::test_django_objecttype_all_fields PASSED                                                                                                                                                        [ 78%]
graphene_django/tests/test_types.py::test_django_objecttype_exclude_fields PASSED                                                                                                                                                    [ 79%]
graphene_django/tests/test_types.py::test_django_objecttype_exclude PASSED                                                                                                                                                           [ 79%]
graphene_django/tests/test_types.py::test_django_objecttype_exclude_fields_and_exclude PASSED                                                                                                                                        [ 80%]
graphene_django/tests/test_types.py::test_django_objecttype_exclude_and_only PASSED                                                                                                                                                  [ 80%]
graphene_django/tests/test_types.py::test_django_objecttype_fields_exclude_type_checking PASSED                                                                                                                                      [ 80%]
graphene_django/tests/test_types.py::test_django_objecttype_fields_exist_on_model PASSED                                                                                                                                             [ 81%]
graphene_django/tests/test_types.py::test_django_objecttype_exclude_fields_exist_on_model PASSED                                                                                                                                     [ 81%]
graphene_django/tests/test_types.py::TestDjangoObjectType::test_django_objecttype_convert_choices_enum_false PASSED                                                                                                                  [ 81%]
graphene_django/tests/test_types.py::TestDjangoObjectType::test_django_objecttype_convert_choices_enum_list PASSED                                                                                                                   [ 82%]
graphene_django/tests/test_types.py::TestDjangoObjectType::test_django_objecttype_convert_choices_enum_empty_list PASSED                                                                                                             [ 82%]
graphene_django/tests/test_types.py::TestDjangoObjectType::test_django_objecttype_convert_choices_enum_naming_collisions PASSED                                                                                                      [ 83%]
graphene_django/tests/test_types.py::TestDjangoObjectType::test_django_objecttype_choices_custom_enum_name PASSED                                                                                                                    [ 83%]
graphene_django/tests/test_utils.py::test_get_model_fields_no_duplication PASSED                                                                                                                                                     [ 83%]
graphene_django/tests/test_utils.py::test_camelize PASSED                                                                                                                                                                            [ 84%]
graphene_django/tests/test_views.py::test_graphiql_is_enabled PASSED                                                                                                                                                                 [ 84%]
graphene_django/tests/test_views.py::test_qfactor_graphiql PASSED                                                                                                                                                                    [ 84%]
graphene_django/tests/test_views.py::test_qfactor_json PASSED                                                                                                                                                                        [ 85%]
graphene_django/tests/test_views.py::test_allows_get_with_query_param PASSED                                                                                                                                                         [ 85%]
graphene_django/tests/test_views.py::test_allows_get_with_variable_values PASSED                                                                                                                                                     [ 86%]
graphene_django/tests/test_views.py::test_allows_get_with_operation_name PASSED                                                                                                                                                      [ 86%]
graphene_django/tests/test_views.py::test_reports_validation_errors PASSED                                                                                                                                                           [ 86%]
graphene_django/tests/test_views.py::test_errors_when_missing_operation_name PASSED                                                                                                                                                  [ 87%]
graphene_django/tests/test_views.py::test_errors_when_sending_a_mutation_via_get PASSED                                                                                                                                              [ 87%]
graphene_django/tests/test_views.py::test_errors_when_selecting_a_mutation_within_a_get PASSED                                                                                                                                       [ 87%]
graphene_django/tests/test_views.py::test_allows_mutation_to_exist_within_a_get PASSED                                                                                                                                               [ 88%]
graphene_django/tests/test_views.py::test_allows_post_with_json_encoding PASSED                                                                                                                                                      [ 88%]
graphene_django/tests/test_views.py::test_batch_allows_post_with_json_encoding PASSED                                                                                                                                                [ 89%]
graphene_django/tests/test_views.py::test_batch_fails_if_is_empty PASSED                                                                                                                                                             [ 89%]
graphene_django/tests/test_views.py::test_allows_sending_a_mutation_via_post PASSED                                                                                                                                                  [ 89%]
graphene_django/tests/test_views.py::test_allows_post_with_url_encoding PASSED                                                                                                                                                       [ 90%]
graphene_django/tests/test_views.py::test_supports_post_json_query_with_string_variables PASSED                                                                                                                                      [ 90%]
graphene_django/tests/test_views.py::test_batch_supports_post_json_query_with_string_variables PASSED                                                                                                                                [ 90%]
graphene_django/tests/test_views.py::test_supports_post_json_query_with_json_variables PASSED                                                                                                                                        [ 91%]
graphene_django/tests/test_views.py::test_batch_supports_post_json_query_with_json_variables PASSED                                                                                                                                  [ 91%]
graphene_django/tests/test_views.py::test_supports_post_url_encoded_query_with_string_variables PASSED                                                                                                                               [ 92%]
graphene_django/tests/test_views.py::test_supports_post_json_quey_with_get_variable_values PASSED                                                                                                                                    [ 92%]
graphene_django/tests/test_views.py::test_post_url_encoded_query_with_get_variable_values PASSED                                                                                                                                     [ 92%]
graphene_django/tests/test_views.py::test_supports_post_raw_text_query_with_get_variable_values PASSED                                                                                                                               [ 93%]
graphene_django/tests/test_views.py::test_allows_post_with_operation_name PASSED                                                                                                                                                     [ 93%]
graphene_django/tests/test_views.py::test_batch_allows_post_with_operation_name PASSED                                                                                                                                               [ 93%]
graphene_django/tests/test_views.py::test_allows_post_with_get_operation_name PASSED                                                                                                                                                 [ 94%]
graphene_django/tests/test_views.py::test_inherited_class_with_attributes_works PASSED                                                                                                                                               [ 94%]
graphene_django/tests/test_views.py::test_supports_pretty_printing PASSED                                                                                                                                                            [ 95%]
graphene_django/tests/test_views.py::test_supports_pretty_printing_by_request PASSED                                                                                                                                                 [ 95%]
graphene_django/tests/test_views.py::test_handles_field_errors_caught_by_graphql PASSED                                                                                                                                              [ 95%]
graphene_django/tests/test_views.py::test_handles_syntax_errors_caught_by_graphql PASSED                                                                                                                                             [ 96%]
graphene_django/tests/test_views.py::test_handles_errors_caused_by_a_lack_of_query PASSED                                                                                                                                            [ 96%]
graphene_django/tests/test_views.py::test_handles_not_expected_json_bodies PASSED                                                                                                                                                    [ 96%]
graphene_django/tests/test_views.py::test_handles_invalid_json_bodies PASSED                                                                                                                                                         [ 97%]
graphene_django/tests/test_views.py::test_handles_django_request_error PASSED                                                                                                                                                        [ 97%]
graphene_django/tests/test_views.py::test_handles_incomplete_json_bodies PASSED                                                                                                                                                      [ 98%]
graphene_django/tests/test_views.py::test_handles_plain_post_text PASSED                                                                                                                                                             [ 98%]
graphene_django/tests/test_views.py::test_handles_poorly_formed_variables PASSED                                                                                                                                                     [ 98%]
graphene_django/tests/test_views.py::test_handles_unsupported_http_methods PASSED                                                                                                                                                    [ 99%]
graphene_django/tests/test_views.py::test_passes_request_into_context_request PASSED                                                                                                                                                 [ 99%]
graphene_django/tests/issues/test_520.py::test_can_use_form_and_serializer_mutations PASSED                                                                                                                                          [100%]

============================================================================================================= warnings summary =============================================================================================================
graphene_django/tests/test_converter.py::test_field_with_choices_convert_enum_false
  /home/ulgens/.pyenv/versions/3.8.2/envs/graphene/lib/python3.8/site-packages/django/db/models/base.py:320: RuntimeWarning: Model 'test.translatedmodel' was already registered. Reloading models is not advised as it can lead to inconsistencies, most notably with related models.
    new_class._meta.apps.register_model(new_class._meta.app_label, new_class)

-- Docs: https://docs.pytest.org/en/latest/warnings.html

----------- coverage: platform linux, python 3.8.2-final-0 -----------
Name                                                     Stmts   Miss  Cover
----------------------------------------------------------------------------
graphene_django/__init__.py                                  4      0   100%
graphene_django/compat.py                                    6      2    67%
graphene_django/converter.py                               171      2    99%
graphene_django/debug/__init__.py                            3      0   100%
graphene_django/debug/middleware.py                         46      5    89%
graphene_django/debug/types.py                               6      0   100%
graphene_django/fields.py                                  113      1    99%
graphene_django/filter/__init__.py                           7      1    86%
graphene_django/filter/fields.py                            39      0   100%
graphene_django/filter/filterset.py                         29      1    97%
graphene_django/filter/utils.py                             24      0   100%
graphene_django/forms/__init__.py                            1      0   100%
graphene_django/forms/converter.py                          52      0   100%
graphene_django/forms/forms.py                              25      2    92%
graphene_django/forms/mutation.py                          102      4    96%
graphene_django/forms/types.py                               2      2     0%
graphene_django/management/__init__.py                       0      0   100%
graphene_django/management/commands/__init__.py              0      0   100%
graphene_django/management/commands/graphql_schema.py       53      7    87%
graphene_django/registry.py                                 23      0   100%
graphene_django/rest_framework/__init__.py                   0      0   100%
graphene_django/rest_framework/models.py                    10      0   100%
graphene_django/rest_framework/mutation.py                  83      0   100%
graphene_django/rest_framework/serializer_converter.py      91      0   100%
graphene_django/rest_framework/types.py                      5      0   100%
graphene_django/settings.py                                 56     12    79%
graphene_django/types.py                                   132      6    95%
graphene_django/utils/__init__.py                            3      0   100%
graphene_django/utils/testing.py                            32     22    31%
graphene_django/utils/utils.py                              48      4    92%
graphene_django/views.py                                   189      8    96%
----------------------------------------------------------------------------
TOTAL                                                     1355     79    94%

====================================================================================================== 266 passed, 1 warning in 5.45s ======================================================================================================
```

Reply:
I just checked Travis and all test combinations were already passing.